app/services/NotificationCache.scala

```diff
@@ -35,9 +35,9 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class DefaultNotificationCache @Inject()(
-  mongoComponent: MongoComponent,
-  appConfig: AppConfig
-)(implicit executionContext: ExecutionContext)
+                                          mongoComponent: MongoComponent,
+                                          appConfig: AppConfig
+                                        )(implicit executionContext: ExecutionContext)
```

Comment:
Can we move the implicit or if there is not enough space there closing )

Reply:
Done